tests/tensor_search/test_validation.py

```diff
@@ -442,12 +449,22 @@ def test_validate_index_settings(self):
             }
         ]
         for settings in good_settings:
-            assert settings == validation.validate_settings_object(settings)
+            assert settings == validate_index_settings(settings,
```

Comment:
Let's move all tests that directly test `validate_index_settings` from marqo commons into the marqo commons repo. That way they are coupled together and there is no need to import any commons errors into marqo test files.